New loc review

Lven-Nemsy opened the pull request:
Got the Modal working, Legend doesn't work yet in it.
Adjusted Navbar - search is aligned with brand name.
Changed search box to Google search.
Renamed the webpage, tried to figure out the navbar- couldn't get it to work.

Nozerone committed:
some development on nav component

RaelNW committed:
imported usestate,usemutate and authentication

RaelNW committed:
Added user signup functionality using useState and Apollo's useMutation

RaelNW committed:
Added handlechange function for form state updates

Nozerone committed:
:g

Nozerone committed:
:g

RaelNW committed:
Added form submission handling with GraphQl mutation and user authentication

briimcfly committed:
added additional data to seeds

RaelNW committed:
Added the modal header and button

RaelNW committed:
Added the modal body and email and username input

RaelNW committed:
Added password input and signup button

RaelNW committed:
added the cancel button

RaelNW committed:
Added console log to handle succes case'

briimcfly committed:
added new TypeDef for Reviews

briimcfly committed:
updated QUERY_LISTINGS

Nozerone committed:
more development on nav bar with some connectivity

Lven-Nemsy committed:
Adding New Location Review

RaelNW committed:
Final commit

briimcfly committed:
Refactor Queries.. Updated Components/Pages

briimcfly committed:
Updated Router for CityAddressList

briimcfly committed:
Updated Path to Location/City/State

briimcfly committed:
Updated Seeds

briimcfly committed:
Updated Model for new Review

briimcfly committed:
updated frontend query for Rentals and Typedefs

briimcfly committed:
updated resolvers

briimcfly committed:
Updated City Rentals to display Rentals

briimcfly committed:
refactored model for Rentals vs Listings

briimcfly committed:
refactored model for Rentals vs Listings

briimcfly committed:
updated router to include RentalReviews

briimcfly committed:
added Rental Review page

briimcfly committed:
added star component

briimcfly committed:
review card styling update

briimcfly committed:
added Loader component

briimcfly committed:
installed google cloud storage

briimcfly committed:
added cloud storage utility

briimcfly committed:
added dotenv

briimcfly committed:
updated AddReview Mutation

briimcfly committed:
added client side mutation for addreview

briimcfly committed:
added imageCarousel.. Images coming from Google Bucket

briimcfly committed:
updated SimpleGrid screen sizes

briimcfly committed:
updated review model to add more ratings

briimcfly committed:
updated query and typedefs for new model

briimcfly committed:
fixed issue with Get Reviews query

briimcfly committed:
updated rental review page to show new ratings

briimcfly committed:
added review atom components

Nozerone committed:
nav component

briimcfly committed:
fixed padding on navbar

briimcfly committed:
updated Dwellex logo to be link

briimcfly committed:
added summary with score context to rental page

briimcfly committed:
updated star styling

briimcfly committed:
added photos to review page

briimcfly committed:
Carousel now clickable on Rental Cards, resolves #20

briimcfly committed:
added separate view button, solves #26

briimcfly committed:
added Add Review Link, solves #28

briimcfly committed:
added comments to Rental Card Component

Nozerone committed:
login working

RaelNW committed:
Added variables for signup and function to open and close the signup modal

RaelNW committed:
Added logic to handle the signup process

RaelNW committed:
added logic to handle the cancel button

Lven-Nemsy committed:
working on newlocationreview

Lven-Nemsy committed:
working on newlocationreview

RaelNW committed:
Added the title text and specified the font size and weight

RaelNW committed:
Added the description text and specified the font size and weight

RaelNW committed:
Added a search bar with an imput placeholder and specified the specs

RaelNW committed:
Removed the login signup and error buttons

RaelNW committed:
Updated the CitySearch jsx so it says go instead of serach and updated the specs so the go button is next to the imput space on the right

RaelNW committed:
Updated the landing page

Lven-Nemsy committed:
modal is now working

Lven-Nemsy committed:
added warning if not logged in

Lven-Nemsy committed:
Adjusted nav bar - everything is aligned properly and replaces search with Google search

Lven-Nemsy committed:
added legend to newlocreview, doesn't work

briimcfly commented:
@Lven-Nemsy i think you need to merge master into your local feature branch. You have merge conflicts that need to be resolved. 

i've been pulling origin master to my local, then switching to my feature branch, then using `git merge main` from my local feature branch then pushing THAT up to my remote .. and its been working out pretty good..

briimcfly reviewed (changes requested):
Github says there's some merge conflicts 
please make sure you're merging master... for some reason your pull request says you're updating 43 files ?? not sureeeee if thats just what git does? But I haven't seen that so far.

Lven-Nemsy commented:
OK, I've been pulling from main to my branch directly. Fixed the merge conflicts, @briimcfly not sure about the 43 files, might have to do with the merge conflicts. I did adjust the Nav bar as appearance wise it was a bit off, and replaced the tab title. 
Still figuring out the questionnaire portion of the review.
